packages/cli/src/main.rs

```diff
@@ -5,6 +5,8 @@
 use std::{env::args_os, ffi::OsStr, path::Path};
 
 fn main() {
+    ctrlc::set_handler(move || eprint!("\x1b[?25h")).expect("fail to set SIGINT handler");
```

Comment:
This should probably be moved into the `run` method in `lib.rs` so it can affect the JS CLI as well.